### PR TITLE
SearchView clears on BackPressed

### PR DESCRIPTION
**Pull request by nitish1211**
This is in reference to #267 

Now the text in the searchview clears on pressing back.

@kunall17 @timabbott  Can you please review this and let me know if need any changes in this.

### CODE REVIEW

**Comment by Sam1301** [comment]
Hi @nitish121, due to recent merges some merge conflicts have occurred, this might help resolve these https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/.

**Comment by nitish1211** [comment]
Has this fixed the issue ?? Or should I add the travis.yml file and create a new pull request for this.

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 1]
```diff
@@ -1457,6 +1458,13 @@ public void run() {
               narrowedList = null;
               getSupportFragmentManager().popBackStack(NARROW,
                     FragmentManager.POP_BACK_STACK_INCLUSIVE);
+
+        } else  if (!searchView.isIconified()) {
```
Comment (Sam1301) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: I am getting a compile time error, this else does not follow an if statement.

**Comment by Sam1301** [comment]
@nitish1211 I would wait for #273 to get merged. Till such time, you can squash commits into one (https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History or http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) and check if your changes solve the issue. Even if you add `.travis.yml` file, you might get lint errors throughout the project, which are also resolved by the mentioned pr.

**Pull request closed without merging**